Code/GraphMol/SmilesParse/CXSmilesOps.cpp

```diff
@@ -1113,6 +1114,71 @@ bool parse_wedged_bonds(Iterator &first, Iterator last, RDKit::RWMol &mol,
   return true;
 }
 
+template <typename Iterator>
+bool parse_doublebond_stereo(Iterator &first, Iterator last, RDKit::RWMol &mol,
```

Comment:
Given that we are passing by value, would that end up changing anything?

Reply:
It wouldn't have meaning for the calling code. It _would_ have meaning within the body of `parse_doublebond_stereo()` - it would ensure that future modifications don't accidentally update the initial value. (as well as communicating that constness to readers of the function).

Basically - it's equivalent to declaring any other variable `const`.